src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -1042,5 +1042,36 @@ describe('toolbar mode only features', () => {
 
       expect(wrapper.findActiveDrawer()!.getElement()).toHaveTextContent('runtime drawer content');
     });
+
+    test('dynamically registered drawers with defaultActive: true should open even when there are already open drawer(s) on the page', async () => {
```

Comment:
Do we have negative case scenarios?

1. After max drawers already open, `defaultActive` does not open a new one
2. After user opened a global drawer manually, late registered `defaultActive` don't open

Reply:
No for both. Do we have any specific requirements for this? If so, could you point me to where I can check them?

Reply:
That's my understanding of our discussion for this bug yesterday. I do not think there is any written down text